exercises/numbers/remainderOf/remainderOf.js

```diff
@@ -15,17 +15,25 @@
  * @param {number} num The input number
  * @returns {boolean} True if num is even and false otherwise
  */
+
 function remainderOf(n, d) {
   /*
```

Comment:
You can remove these comments. They're just here for instructional purposes, but add noise to the code when reviewing.